public/stylesheets/scss/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$font-light: #B4C4CF;
```

Comment:
Interesting. That's definitely another way of looking at it. I guess I was thinking in terms of the general palette, but it seems to me that this'll work for now.